components/Pages/CarePackages/ReviewPackageDetails/ReviewPackageDetailsButtons.js

```diff
@@ -0,0 +1,12 @@
+import { Button, Container } from '../../../HackneyDS';
+import React from 'react';
+
+const ReviewPackageDetailsButtons = ({ edit, cancel, end }) => (
+  <Container className="review-package-details__buttons">
+    <Button className='outline blue' handler={edit}>Edit</Button>
+    <Button className='outline red' handler={cancel}>Cancel</Button>
+    <Button className='outline blue' handler={end}>End</Button>
+  </Container>
+);
+
+export default ReviewPackageDetailsButtons;
```

Comment:
Better to always wrap component with `memo()`. It will prevent redundant rerenders